english/craving/craving.md

```diff
@@ -0,0 +1 @@
+You are craving marshmellows covered in chocolate! You can't believe how powerful the craving is! You must have them now! You quickly put some in a bowl and cover them with Chocolate. [You begin eating!](eating-walls/eating-marshmellows.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.